src/main/kotlin/net/maizegenetics/phgv2/cli/StartServer.kt

```diff
@@ -1,21 +1,104 @@
 package net.maizegenetics.phgv2.cli
 
 import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.default
+import com.github.ajalt.clikt.parameters.options.option
 import io.ktor.server.engine.*
 import io.ktor.server.netty.*
 import net.maizegenetics.phgv2.utils.setupDebugLogging
+import net.maizegenetics.phgv2.utils.verifyURI
 import org.apache.logging.log4j.LogManager
+import java.nio.file.Files
+import java.nio.file.Paths
 
 class StartServer : CliktCommand(help = "Starts PHGv2 BrAPI Server") {
 
     private val myLogger = LogManager.getLogger(StartServer::class.java)
+    val dbPath by option(help = "Full path to folder where TileDB datasets are stored.  \nThis must be run at least once before starting the server. \nIf you have already run it for this server instance you need not supply this again.")
```

Comment:
I had to read the help message twice before I realized I was missing the "not" in "need not". Maybe change to "you do not need to supply this again" for clarity.

Reply:
Changed as suggested